services/mining/newblocktemplate.go

```diff
@@ -491,15 +443,7 @@ mempoolLoop:
 		Height:          nextBlockHeight,
 		Blues:           blues,
 		ValidPayAddress: payToAddress != nil,
-		PowDiffData: types.PowDiffStandard{
-			Blake2bDTarget:         reqBlake2bDDifficulty,
-			X16rv3DTarget:          reqX16rv3Difficulty,
-			X8r16DTarget:           reqX8r16Difficulty,
-			QitmeerKeccak256Target: keccak256Difficulty,
-			CuckarooBaseDiff:       pow.CompactToBig(reqCuckarooDifficulty).Uint64(),
-			CuckaroomBaseDiff:      pow.CompactToBig(reqCuckaroomDifficulty).Uint64(),
-			CuckatooBaseDiff:       pow.CompactToBig(reqCuckatooDifficulty).Uint64(),
-		},
+		CompactDiff:     reqCompactDifficulty,
```

Comment:
If there are not different meaning for compact difficulty and uncompact difficulty, I think the naming should be be consist with `BlockHeader`, using different naming might encourage the guess that where are two kinds of difficulties. 
```go
types.BlockTemplate{
	CompactDiff: reqCompactDifficulty,
}

types.BlockHeader{
	Difficulty: reqCompactDifficulty,
}
```